axlearn/open_api/gemini.py

```diff
@@ -188,6 +187,24 @@ def _aggregate_tool_role_messages(messages: List[Dict[str, Any]]) -> List[Dict[s
     return aggregated_messages
 
 
+def _santize_request(request: Dict[str, Any]):
+    """Santizes request to follow Gemini request rules."""
```

Comment:
```suggestion
def _format_request(request: Dict[str, Any]):
    """Formats request to follow Gemini request rules."""
```
Since it seems more like formatting than sanitization?

Reply:
changed